pkg/cmd/testhelpers/cmd_test_helpers.go

```diff
@@ -149,37 +151,20 @@ func MockFactoryFakeClients(mockFactory *clients_test.MockFactory) {
 func CreateTestJxHomeDir() (string, string, error) {
```

Comment:
Good change looking good now. Could you do me another favour and adjust the Godoc, getting rid of "copying over any existing config"

Reply:
Sure. Thanks.